about.html

```diff
@@ -0,0 +1,46 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Me</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">My Portfolio</a></li>
+                <li><a href="contact.html">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <h1>My Portfolio</h1>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed et metus ac enim suscipit volutpat. Sed ac leo
+            nec justo bibendum pulvinar.</p>
+
+            <div class ="projects">
```

Comment:
great use of semantic elements in this and other pages! consider seeing where else you can increase accessibility through additional semantic elements: https://www.w3schools.com/html/html5_semantic_elements.asp